demos/src/simple.mustache

```diff
@@ -1,3 +1,37 @@
+<h1>Button toggle</h1>
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="2" data-o-expander-count-selector="li">
+	<h2>Collapsing to number of items in a list</h2>
+	<ul class="o-expander__content">
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+	</ul>
+	<button class="o-expander__toggle o--if-js"></button>
+</div>
+
+<div data-o-component="o-expander" class="o-expander height" data-o-expander-shrink-to="height">
+	<h2>Collapsing to height of content (resize window to see toggle appear and disappear in a content-aware manner)</h2>
+	<div class="o-expander__content">
+		word word word word word word word word word word word word word word word word word word word word word word
+	</div>
+	<button class="o-expander__toggle o--if-js"></button>
```

Comment:
The text is added with the JS
![screen shot 2016-09-22 at 11 49 16](https://cloud.githubusercontent.com/assets/3081639/18745592/a18a7aa4-80ba-11e6-9992-041fa568cc98.png)